dashboard/src/components/TooltipDateTime/TooltipDateTime.tsx

```diff
@@ -0,0 +1,56 @@
+import { format, isValid } from 'date-fns';
+
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/Tooltip';
+import { getDateOffset } from '@/utils/utils';
+
+type TooltipDateTimeProps = {
+  dateTime: string;
+  dateFormat?: string;
+  timeFormat?: string;
+  lineBreak?: boolean;
+  showLabelTime?: boolean;
+  showLabelTZ?: boolean;
+  showTooltip?: boolean;
+};
+
+const TooltipDateTime = ({
+  dateTime,
+  dateFormat,
+  timeFormat,
+  lineBreak,
+  showLabelTime,
+  showLabelTZ,
+  showTooltip,
```

Comment:
The majority of the use will be with `lineBreak`, `showLabelTZ` and `showTooltip` so we can let this values as `true` by default

```suggestion
  lineBreak,
  showLabelTime,
  showLabelTZ,
  showTooltip,
```

```suggestion
  lineBreak = true,
  showLabelTime,
  showLabelTZ = true,
  showTooltip = true,
```
and then just use 
```
<TooltipDateTime dateTime={test.startTime} />
```